tests/lib/cli.js

```diff
@@ -1112,4 +1112,12 @@ describe("cli", () => {
         });
     });
 
+    describe("when passing --info", () => {
+        it("should print out debugging information concerning the local environment", () => {
+            cli.execute("--info");
+
+            assert.isTrue(log.info.calledOnce);
```

Comment:
It looks like this test is running synchronously, so I think it'll pass after the first `log.info` on [line 187](https://github.com/eslint/eslint/pull/12060/files#diff-f8849ae9f0614ba9464fdd14c2d98dc0R187) without actually testing that the environment info is actually printed out.